certified-connectors/dvelop/apiDefinition.swagger.json

```diff
@@ -835,6 +835,474 @@
+                "operationId": "DefaultGetRequestSchema",
+                "value-path": "request",
+                "parameters": {
+                  "defaultAction": "updatedocument",
+                  "paramDef": "k1:repositoryId|k2:docTypeId",
+                  "k1": {
+                    "parameter": "repositoryId"
+                  },
+                  "k2": {
+                    "parameter": "categoryId"
+                  },
+                  "k3": "dummy",
+                  "k4": "dummy",
+                  "k5": "dummy"
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "DefaultGetRequestSchema",
+                "parameters": {
+                  "defaultAction": {
+                    "value": "updatedocument"
+                  },
+                  "paramDef": {
+                    "value": "k1:repositoryId|k2:docTypeId"
+                  },
+                  "k1": {
+                    "parameterReference": "repositoryId"
+                  },
+                  "k2": {
+                    "parameterReference": "categoryId"
+                  },
+                  "k3": {
+                    "value": "dummy"
+                  },
+                  "k4": {
+                    "value": "dummy"
+                  },
+                  "k5": {
+                    "value": "dummy"
+                  }
+                },
+                "itemValuePath": "request"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response in case of a successful document update.",
+            "schema": {
+              "x-ms-dynamic-schema": {
+                "operationId": "DefaultGetResponseSchema",
+                "parameters": {
+                  "defaultAction": "updatedocument",
+                  "k1": "dummy",
+                  "k2": "dummy",
+                  "k3": "dummy",
+                  "k4": "dummy",
+                  "k5": "dummy"
+                },
+                "value-path": "response"
+              }
+            }
+          },
+          "403": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "default": {
+            "$ref": "#/responses/DefaultError"
+          }
+        }
+      }
+    },
+    "/default/createdossier/execute": {
+      "post": {
+        "tags": [
+          "Default actions"
+        ],
+        "summary": "Create dossier",
+        "description": "Create a dossier in d.velop documents.",
+        "operationId": "CreateDossier",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "repositoryId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Select an repository of your d.velop documents environment.",
```

Comment:
@BeritHorst please make sure that everything is grammatically correct and is free of typographical errors. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).